pkg/stub/imagestreams.go

```diff
@@ -3,6 +3,7 @@ package stub
 import (
 	"context"
 	"fmt"
+	"github.com/openshift/cluster-samples-operator/pkg/metrics"
```

Comment:
This import got organized into the incorrect group.

Reply:
part of next push